ext/dcos-installer/dcos_installer/action_lib/configure.py

```diff
@@ -44,20 +56,19 @@ def do_validate_gen_config(gen_config):
     return gen.validate(arguments=gen_config)
 
 
-def fetch_bootstrap(bootstrap_id):
+def fetch_bootstrap(artifact_dir, dest_dir, bootstrap_id):
```

Comment:
That code is setup so that we "atomically" copy things out as best as possible, just in case the destination device becomes completely full or something similar people don't end up with partially extracted files (With the bootstrap tarball it did actually happen a couple of times).

We really should probably extend fetch_bootstrap to do all the cluster packages as well, they need the same "atomic" copy out guarantees.